app/controllers/events_controller.rb

```diff
@@ -20,6 +20,7 @@ def show
 
     respond_to do |format|
       format.html # show.html.erb
+      format.json { render :json => @event.attributes.merge({ attendance_options: @event.attendance_options }) }
```

Comment:
Line is too long. [112/80]<br>Redundant curly braces around a hash parameter.<br>Use the new Ruby 1.9 hash syntax.